### PR TITLE
Deprecate -moz-image-rect function

### PR DESCRIPTION
**Pull request by pepelsbey**
### Description

Deprecated docs for non-standard `-moz-image-rect` function for clipping background images.

### Motivation

Removal in the upcoming Firefox 120 release.

### Related issues and pull requests

Part of the Firefox 120 release project #29784

### CODE REVIEW

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Intervening while this is still in draft (I don't do this usually) because this is not going in the right direction.

MDN usage (that follows BCD rules) is to remove the main information about a deleted feature only two years after its deletion.

This timing was chosen because this feature can stay in the ESR version of Firefox for up to 60 weeks, with some extra months to allow people dealing with bugs in websites to find the information about the intended purpose of the deleted feature they have to replace.

In the specific case here, this feature will be available in a supported ESR Firefox until October 2024 at least.

Of course, there is information to be updated now.

As much as I'm happy to see non-standard stuff be removed from browsers, deleting this page is too early. Note that we won't forget to delete it in 2025 as BCD notifies us when a feature has been removed for two years.

**Comment by pepelsbey** [comment]
@teoli2003 thanks! Got a heads-up from @bsmth already. It’s now deprecation PR. Sorry, learning on the fly 😅

**Comment by teoli2003** [comment]
> @teoli2003 thanks! Got a heads-up from @bsmth already. It’s now a deprecation PR. Sorry, learning on the fly 😅

No worry, that's why my guess 😃 . That's why I took the time to write down the background reason!

File: files/en-us/web/css/-moz-image-rect/index.md [hunk 1]
```diff
@@ -3,12 +3,13 @@ title: "-moz-image-rect"
 slug: Web/CSS/-moz-image-rect
 page-type: css-function
 status:
+  - deprecated
```
Comment (teoli2003) [inline files/en-us/web/css/-moz-image-rect/index.md:6]: This is ok but will go away as these status flags are set by a script reading BCD. (So you'll have to add it to the PR there)

File: files/en-us/web/css/-moz-image-rect/index.md [hunk 2]
```diff
@@ -3,12 +3,13 @@ title: "-moz-image-rect"
...
   - experimental
   - non-standard
 browser-compat: css.types.-moz-image-rect
 ---
 
-{{CSSRef}}{{Non-standard_Header}}{{SeeCompatTable}}
+{{CSSRef}}{{Non-standard_Header}}{{Deprecated_Header}}
```
Comment (teoli2003) [inline files/en-us/web/css/-moz-image-rect/index.md:12]: Same here (fine but will go away when the script runs)

**Review by teoli2003 — APPROVED** [review]
I'll leave this open for others to review. My objection was about the deletion of the page.

File: files/en-us/web/css/-moz-image-rect/index.md [hunk 3]
```diff
@@ -3,12 +3,13 @@ title: "-moz-image-rect"
 slug: Web/CSS/-moz-image-rect
 page-type: css-function
 status:
+  - deprecated
   - experimental
```
Comment (teoli2003) [inline files/en-us/web/css/-moz-image-rect/index.md:7]: I guess the "experimental" will have to go away (but it will be enough to do it in the BCD PR)

**Comment by pepelsbey** [comment]
@teoli2003 I removed the `experimental` flag and filed and filed a [deprecation PR](https://github.com/mdn/browser-compat-data/pull/21268) to BCD as well. I think it’s fine if those flags will be updated by the script later.

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Pull request merged**